src/masoniteorm/commands/MakeModelCommand.py

```diff
@@ -16,6 +16,7 @@ class MakeModelCommand(Command):
         {--c|create : If the migration file should create a table}
         {--t|table : If the migration file should modify an existing table}
         {--d|directory=app : The location of the model directory}
+        {--D|migrations-directory=databases/migrations : The location of the migration directory}
```

Comment:
This is the same as -d right? what is -D?

Reply:
never I see lol

Reply:
The issue with Cleo commands is that the short name of the option needs to be only one letter... I cannot use `md` for migrations directory...so either you have a better idea for the letter, either I went with the uppercased version.

Reply:
It's already the case with the config option which is using `-C` because `-c` is taken by connection option

Reply:
No worries I thought you were doing the model directory but I read "model directory" instead of "migration directory"